### ISSUE TITLE
[MPLUGIN-517] GoalRenderer renderParameterDetails() renders in wrong order

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-plugin-tools#817]
**[Max Philipp Wriedt](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=JIRAUSER299058)** opened **[MPLUGIN-517](https://issues.apache.org/jira/browse/MPLUGIN-517?redirect=false)** and commented

GoalRenderer.java:360

```java
            sink.anchor(parameter.getName());

            startSection(format("parameter.name", parameter.getName()));
            sink.anchor_();
```

The closing `anchor_();` should be before starting the section.

---

**Affects:** 3.11.0, 3.12.0

**Issue Links:**
- [TORQUE-367](https://issues.apache.org/jira/browse/TORQUE-367) Maven 2 Plugin Report is incorrectly rendered
 (_**"blocks"**_)

**Remote Links:**
- [GitHub Pull Request #277
](https://github.com/apache/maven-plugin-tools/pull/277)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

That one slipped through...will fix.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [e746d9cc35cb03c03c61101d2b11f8247e6dcc82](https://gitbox.apache.org/repos/asf?p=maven-plugin-tools.git&a=commit&h=e746d9cc35cb03c03c61101d2b11f8247e6dcc82).
